ansible/roles/accumulo/templates/tserver.j2

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=TServer Service for Accumulo
+Requires=network.target 
+After=network.target
+
+[Service]
+User={{ cluster_user }}
+Group={{ cluster_group }}
+Type=simple
+ExecStart={{ accumulo_home }}/bin/accumulo tserver{% if accumulo_major_version == '1' %}-{{ tserver_instance_id }}{% endif %}{{''}}
+Environment=ACCUMULO_HOME={{ accumulo_home }}
+Environment=JAVA_HOME={{ java_home }}
+{% if accumulo_major_version == '2' %}
+Environment=ACCUMULO_SERVICE_INSTANCE={{ tserver_instance_id }}
+{% endif %}
```

Comment:
If systemd templating is used, you don't need to use the ansible templating here for the tserver_instance_id. You can instead use `%I` (or `%i`):

https://fedoramagazine.org/systemd-template-unit-files/

Reply:
done